ideascube/mediacenter/tests/test_views.py

```diff
@@ -620,3 +620,27 @@ def test_can_update_document_tags(staffapp, pdf):
     doc = Document.objects.get(pk=pdf.pk)
     assert doc.tags.count() == 2
     assert doc.tags.first().name == 'tag1'
+
+
+def test_content_summary_is_not_cleaned_from_wanted_html_tags(staffapp):
+    url = reverse('mediacenter:document_create')
+    form = staffapp.get(url).forms['model_form']
+    form['title'] = 'my content title'
+    form['summary'] = 'my content summary <a href="foo">link</a>'
+    form['credits'] = 'my document credits'
+    form['original'] = Upload('image.jpg', b'xxxxxx', 'image/jpeg')
+    form.submit().follow()
+    document = Document.objects.first()
+    assert document.summary == ('my content summary <a href="foo">link</a>')
```

Comment:
There is no javascript here, so no tinyMCE. Only bleach is handling this content.
But yes, if a user would have enter this text in its browser, tinyMCE would have sent us the content wrapper in a `<p>`.
For consistency, I should change the test string to add the `<p>...</p>` tags.